requirements.txt

```diff
@@ -8,8 +8,6 @@ click==7.0
 # this is the pipelines version running in 2.8/2.10 iguazio system, for now locking it to this version
 kfp==0.2.5
 nest-asyncio>=1.0.0
-# <0.16 from botocore inside nuclio-jupyter, >=0.13.1 from readme-renderer inside twine (from dev-requirements)
-docutils<0.16, >=0.13.1
```

Comment:
Remove the first `# If you change this file, run "make docs-requirements"` line (I can't comment on a line you didn't change)

Reply:
Looking on the comment, `nuclio-jupyter` is still here, and `twine` is still in `dev-requirements.txt` so I think this is still needed here, let's keep here and remove from docs requirements